client/me/profile-links-add-wordpress/index.jsx

```diff
@@ -172,7 +170,9 @@ class ProfileLinksAddWordPress extends Component {
 							components: {
 								jetpackLink: (
 									<a
-										href="#"
+										href="http://jetpack.me"
```

Comment:
I know this was like this before, but since we're touching it already, would you mind making it `https`?